util_test.ts

```diff
@@ -173,13 +172,6 @@ Deno.test("homeDir", async ({ step }) => {
       Deno.env.delete("HOME");
     });
   }
-
-  await step("unreachable", () => {
-    // @ts-ignore TS2540
-    Deno.build = { os: "freebsd" };
-    assertThrows(() => homeDir(), Error, "unreachable");
-  });
```

Comment:
We still want to test this. Just set the OS to anything and make sure it still throws unreachable for unknown os's.